
>>> i = new Main::A
wrong-args.hlt:23-24: error, operands do not match instruction new
  got:
    new: Main::A -> int<64>
  expected one of:
    new: bytes -> ref<bytes>
    new: callable<*>, <hook>, tuple<*> -> ref<callable<*>>
    new: callable<*>, function() -> void, tuple<*> -> ref<callable<*>>
    new: channel<*>, [ int<64> ] -> ref<channel<*>>
    new: classifier<*> -> ref<classifier<*>>
    new: exception -> ref<exception>
    new: exception, any -> ref<exception>
    new: file -> ref<file>
    new: iosrc<*>, string -> ref<iosrc<*>>
    new: list<*>, [ ref<timer_mgr> ] -> ref<list<*>>
    new: map<*>, [ ref<timer_mgr> ] -> ref<map<*>>
    new: regexp<> -> ref<regexp<>>
    new: set<*>, [ ref<timer_mgr> ] -> ref<set<*>>
    new: struct { }  -> ref<struct { } >
    new: timer, function() -> void, tuple<*> -> ref<timer>
    new: timer_mgr -> ref<timer_mgr>
    new: vector<*>, [ ref<timer_mgr> ] -> ref<vector<*>>
 [pass::hilti::InstructionResolver]
wrong-args.hlt: Aborting due to verification error.
error running hiltic, aborting
